_data/projects/Labforum.yml

```diff
@@ -0,0 +1,12 @@
+name: Labforum
+desc: A lightweight, API powered forum engine that deploys quickly and have cross-platform support.
+site: https://www.labstry.com
+tags:
+  - php
+  - web
+  - html
+  - cross-platforum
+  - forum
+upforgrabs:
+  name: help wanted
+  link: https://github.com/labstry-opensource/labforum
```

Comment:
@littlephone Please change this to https://github.com/labstry-opensource/labforum/labels/help%20wanted

Reply:
@ritwik12 The change is completed.